include/deal.II/base/vectorization.h

```diff
@@ -2854,10 +2874,10 @@ class VectorizedArray<float>
  */
```

Comment:
Please adjust friend declarations in lines 2856-2867 above (and in the other specializations if appropriate). The CI tests of clang-tidy and on OSX complain about a private `get_abs()` function.